defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+xen_vman_vms:                   []
+xen_vman_memory:                1024
+xen_vman_vcpus:                 2
+xen_vman_auto_boot:             1
+xen_vman_storage_type:          nfs
+xen_vman_cpu_str:               all,^1
+xen_vman_spice:                 0
+xen_vman_spicehost:             0.0.0.0
+xen_vman_spiceport:             3100
+xen_vman_mouse:                 1
+xen_vman_spicevdagent:          1
+xen_vman_share_clipboard:       1
+xen_vman_max_usb_redirections:  4
+xen_vman_vm_config_path:        /etc/xen/vms
+xen_vman_disks:                 []
+xen_vman_builder:               generic
+xen_vman_additonal_xen_options: []
+xen_vman_start_timeout:         15
+xen_vman_stop_timeout:          300
+xen_vman_iscsi_login_timeout:   40
+xen_vman_iscsi_logout_timeout:  40
+xen_vman_default_filesystem:    ext4
```

Comment:
Nice that you indented **almost** all lines on the same level

Reply:
What you mean with almost?

Reply:
He meant that all lines except this one are indented correctly.